packages/rsocket-tcp-client/src/index.ts

```diff
@@ -0,0 +1,19 @@
+/** Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
should we just replace these license headers with the one in rsocket java?

Reply:
Woops, yeah ill remove this license header.

Is it necessary to have license headers in every file? Most commonly it seems that projects just have a LICENSE file in their root.

Reply:
we can sort that out a bit later. Can be skip now